src/page-content/content-cloud.tsx

```diff
@@ -31,18 +31,18 @@ export const hero_alt: HeroImageProps = {
   id: 'cloud.hero',
   title: (
     <>
-      <span className={'is-themed-primary'}> Cloud native </span> identity
+      <span className={'is-themed-primary'}> Ory Cloud </span> uses idustry standards
     </>
   ),
   description: (
     <>
-      Integrate modern identity services with Ory Cloud. Batteries included.
-      <span className={'is-themed-primary'}> Based on Ory Open Source. </span>
+      Ory Cloud is API-first, customizable, privacy conform and secure and deploys all Ory services in concert - at any scale, across the globe.
```

Comment:
```suggestion
      Ory Cloud is API-first, customizable, privacy conform, secure, and deploys all Ory services - at any scale, across the globe.
```

not sure what is meant by `in concert`